test/integration/test.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-docker run --net=app_default fleet-telemetry-integration-tests
+docker run --net=app_default --name integration-test --rm fleet-telemetry-integration-tests
```

Comment:
nit: why was this change required?

Reply:
> Additional Notes
> 
> I am naming the integration tests container so it can be referenced through Docker DNS.

Instead of having a container running for receiving messages (like kafka and pubsub), the integration test creates a web server on port 3000 to receive events over http. To reference this server, we need a hostname, thus naming this `integration-test`.

[See here](https://github.com/teslamotors/fleet-telemetry/pull/91/files#diff-de5feecb6b65fc15b683b1ce31b1c022fe52a45d66dd81c501aeeb79c5703773R30)